CompanyEmployees.Core.Domain/Exceptions/AttendanceAlreadyExistsException.cs

```diff
@@ -0,0 +1,9 @@
+namespace CompanyEmployees.Core.Domain.Exceptions;
+
+public sealed class AttendanceAlreadyExistsException : BadRequestException
```

Comment:
For consistency and clarity, you may want to explicitly import System and other necessary namespaces since you're using Guid. While not required here, it improves future maintainability.

```suggestion
using System;
```